src/api/rest/providers/github/client.js

```diff
@@ -0,0 +1,132 @@
+import { Client } from '../base';
+import Schemas from './schemas';
+
+export class Github extends Client {
+  API_ROOT = 'https://api.github.com/';
+
+  setAccessToken = accessToken => {
+    this.authHeaders = { Authorization: `token ${accessToken}` };
+  };
+
+  getNextPageUrl = response => {
+    const link = response.headers.get('link');
+
+    if (!link) {
+      return null;
+    }
+
+    const nextLink = link.split(',').find(s => s.indexOf('rel="next"') > -1);
+
+    if (!nextLink) {
+      return null;
+    }
+
+    return nextLink.split(';')[0].slice(1, -1);
+  };
+
+  /**
+   * The organizations endpoint
+   */
+  orgs = {
+    /**
+     * Gets an organization by its id
+     *
+     * @param {string} orgId
+     */
```

Comment:
Description for `params` will be the same for all methods. I wanted to avoid duplicating doc comments before finding a good name.

The JSDoc is in Github for now, but I'll move it to the Rest class for later. (We're going to add all the blanks prototypes there along with the JSDocs to be able to generate an API documentation like this: https://octokit.github.io/node-github/#api-activity-markNotificationThreadAsRead)

Does that sound ok to you?